boxoffice/templates/order.html

```diff
@@ -24,115 +25,138 @@ <h1 class="header">{{order.item_collection.title}} Ticket Assignment</h1>
       {%- raw %}
         <script id='boxoffice-ticket-template' type='text/ractive'>
           {{#line_items:line_item}}
-            <div class="ticket col-sm-6 col-xs-12" id="item-{{ seq }}">
-              <div class="heading">
-                <div class="ticket-type">
-                  <p>{{ title }}</p>
-                </div>
-                <div class="ticket-edit">
-                  {{#if assignee.id || isTicketAssigned}}
-                    <span><i class="fa fa-user"></i><i class="fa fa-check"></i></span>
-                    <span class="edit-btn" on-click="assign(event, event.keypath, true)"><i class="fa fa-edit"></i></span>
-                  {{else}}
-                    <span><i class="fa fa-user"></i><i class="fa fa-question"></i></span>
-                  {{/if}}
-                </div>
-              </div>
-              <div class="content">
-                {{#if !toAssign}}
-                  <div class="content-box clearfix" intro='fly:{"x":20,"y":"0"}'>
-                    <div class="item-description">{{{ description }}}</div>
-                    <p class="price center">&#8377;{{ final_amount }}</p>
+            {{#if !is_cancelled}}
```

Comment:
Checking for explicit states makes the code more resilient to new states being introduced.